helmfile/values/user-rbac.yaml.gotmpl

```diff
@@ -1,4 +1,5 @@
 namespaces: {{ toYaml .Values.user.namespaces | nindent 2 }}
 users: {{ toYaml .Values.user.adminUsers | nindent 2 }}
+create_namespaces: {{ .Values.user.createNamespaces }}
```

Comment:
Can we, while we're here and editing, change this to `createNamespace`?

Reply:
Fixed now